packages/web3torrent/notes/channels-integration.md

```diff
@@ -0,0 +1,45 @@
+# How are channels integrated into web3Torrent?
+
+Web3Torrent uses state channels tech to stream money between webTorrent peers, so that downloaders may pay uploaders for data. The granularity of the payments can be tuned, which alters the trust-performance tradeoff. For example, peers can pay per byte (low trust, high overhead for processing payments resulting in low data transfer speeds) or per Megabyte (higher trust, lower overhead and greater transfer speed).
+
+These tradeoffs are set in [`constants.ts`](https://github.com/statechannels/monorepo/blob/master/packages/web3torrent/src/constants.ts).
+
+## Payment channel client
+
+Web3Torrent includes a general purpose, 2-party unidirectional payment channel client that wraps the state channels wallet provider and provides convenient methods such as `sendPayment`. This client takes care of all downstream encoding to interface with the state channels stack.
+
+## Paid streaming extension
+
+Web3Torrent consists of a paid streaming extension to the webTorrent protocol. The purpose of the extension is
```

Comment:
I think the precise term should be an extension of [BitTorrent protocol](https://webtorrent.io/docs).

Reply:
Hopefully clearer in 91fad3d